web/forms.py

```
@@ -1768,3 +1773,42 @@ class StudyGroupForm(forms.ModelForm):
+            "slug",
+            "description",
+            "features",
+            "price_monthly",
+            "price_yearly",
+            "billing_period",
+            "stripe_monthly_price_id",
+            "stripe_yearly_price_id",
+            "is_active",
+            "is_popular",
+            "order",
+        ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)